frontend/src/components/pages/camps/CampersTableNavBar.tsx

```diff
@@ -0,0 +1,138 @@
+import CampersTable from "./CampersTable";
+
+const CampersTableNavBar = () => {
+  const [tabIndex, setTabIndex] = React.useState(0);
+
+  const campCapacity = 20;
+  const emptyTest: Camper[] = [];
+  const campersTest: Camper[] = [
+    {
+      firstName: "Joe",
+      lastName: "Bob",
+      age: 12,
+      allergies: "you",
+      earlyDropoff: [new Date()],
+      latePickup: [new Date()],
+      specialNeeds: "",
+      contacts: [
+        {
+          firstName: "Mommy",
+          lastName: "Bob",
+          email: "mommyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+        {
+          firstName: "Daddy",
+          lastName: "Bob",
+          email: "daddyyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+      ],
+    },
+    {
+      firstName: "Thing",
+      lastName: "Bob",
+      age: 10,
+      allergies: "you",
+      earlyDropoff: [new Date()],
+      latePickup: [new Date()],
+      specialNeeds: "yes",
+      contacts: [
+        {
+          firstName: "Mommy",
+          lastName: "Yot",
+          email: "mommyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+        {
+          firstName: "Daddy",
+          lastName: "Stuff",
+          email: "daddyyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+      ],
+    },
+    {
+      firstName: "Bro",
+      lastName: "Bob",
+      age: 11,
+      allergies: "you",
+      earlyDropoff: [new Date()],
+      latePickup: [new Date()],
+      specialNeeds: "",
+      contacts: [
+        {
+          firstName: "Mommy",
+          lastName: "Yes",
+          email: "mommyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+        {
+          firstName: "Daddy",
+          lastName: "That",
+          email: "daddyyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+      ],
+    },
+    {
+      firstName: "Sup",
+      lastName: "Bob",
+      age: 9,
+      allergies: "you",
+      earlyDropoff: [new Date()],
+      latePickup: [new Date()],
+      specialNeeds: "",
+      contacts: [
+        {
+          firstName: "Mommy",
+          lastName: "Nice",
+          email: "mommyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+        {
+          firstName: "Daddy",
+          lastName: "Yoyo",
+          email: "daddyyyy@gmail.com",
+          phoneNumber: "123-456-7890",
+        },
+      ],
+    },
+  ];
+
+  return (
+    <Container maxW="90vw" width="100%">
+      <Tabs
+        onChange={(index) => setTabIndex(index)}
+        variant="line"
+        colorScheme="green"
+        outline="0"
+      >
+        <TabList>
+          <Tab whiteSpace="nowrap">Registration List</Tab>
+          <Tab>Waitlist</Tab>
+        </TabList>
+        <TabPanels width="100%" marginTop="16px">
+          <TabPanel padding="0">
+            <CampersTable campers={campersTest} campCapacity={campCapacity} />
```

Comment:
You would pass in the selected session's camper list and camp capacity here right? The table would need to render the data for the selected session. Could you leave a TODO comment to make this dependant on the session data here?